### ISSUE TITLE
Do we really need a character approval?

### ISSUE THREAD

**Issue by zmoazeni** [our-stories/west-marches#29]
Could we just have a detailed checklist in the character template to help the person vet the concept?

**Comment by crimsonknave** [comment]
I don't think we need anything complicated, but I think it would be nice to have an official "I've read this and I don't have any questions" stamp.

**Comment by djigneo** [comment]
I also think there should be some degree of approval on a character concept before the character sheet is drafted. Something to guide character motivation mainly. Here's the document my group drafted to help with character creation. Some parts reference static parties (building a character that fits in), but I think the other things are good things to think about when working on a concept.
[Step by step characters.docx](https://github.com/our-stories/west-marches/files/5038804/Step.by.step.characters.docx)

**Comment by zmoazeni** [comment]
Right on. That makes sense. We can have the pool of GMs vet it.

I think I might checklist-ify both character and session issues to make it easier to know what step you're in.

**Comment by zmoazeni** [comment]
@djigneo This content in your docx is awesome. Do you mind if I copy the contents to a file in the repository with some of my flair?

**Comment by djigneo** [comment]
@zmoazeni I do not mind.

**Comment by djigneo** [comment]
Self-assigning to create a markdown file for this.

**Comment by djigneo** [comment]
[Markdown file](https://github.com/our-stories/west-marches/blob/main/character%20creation%20guidelines.md) created. Suggest closing this issue.

**Comment by zmoazeni** [comment]
I think we need to update the issue template before we can close this. The process is still scattered. 

That guideline can definitely be linked from there.

I also commented on the commit itself. There's a lot of vague language that I think we can clean up by iterating it in a pull request. 

Can you revert those commits to master and start it as a PR?

**Comment by zmoazeni** [comment]
Reopening, I don't think we've fully settled on a conclusion here.
